docs/dashboard/configuration/attributes.md

```diff
@@ -2,5 +2,95 @@
 id: attributes
 title: Attributes
 ---
+## About Attributes
+
+The Attributes page allows you to use the full capacity of the product structure in Saleor. By configuring your products right, product management in Saleor becomes an hassle-free task.
+
+The main Attributes page displays a list of all existing attributes. The list consist of several columns indicating the attributes' parameters (such as an attribute code or whether the attribute is visible to your customers).
+
+You can create variations of products you sell to reflect the wide range of colours, sizes or material the product was made from.
```

Comment:
"colour" should be changed to "color" as we use American English in Saleor. There are a few more occurrences to change in this file.